# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: \n"
"POT-Creation-Date: 2017-08-26 18:11+EDT\n"
"PO-Revision-Date: 2017-08-27 09:02-0600\n"
"Language-Team: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: pygettext.py 1.5\n"
"X-Generator: Poedit 2.0.3\n"
"Last-Translator: UltimatePancake <pier.gaetani@gmail.com>\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"Language: es\n"

#: ../cog_manager.py:196
msgid ""
"Install Path: {}\n"
"\n"
msgstr ""
"Ubicación de instalación: {}\n"
"\n"

#: ../cog_manager.py:212
msgid "That path is does not exist or does not point to a valid directory."
msgstr "Esa ubicación no existe o no apunta a un directorio válido."

#: ../cog_manager.py:221
msgid "Path successfully added."
msgstr "Ubicación agregada exitósamente."

#: ../cog_manager.py:234
msgid "That is an invalid path number."
msgstr "Número de ubicación inválido."

#: ../cog_manager.py:238
msgid "Path successfully removed."
msgstr "Ubicación eliminada exitósamente."

#: ../cog_manager.py:254
msgid "Invalid 'from' index."
msgstr "Índice 'from' inválido."

#: ../cog_manager.py:260
msgid "Invalid 'to' index."
msgstr "Índice 'to' inválido."

#: ../cog_manager.py:264
msgid "Paths reordered."
msgstr "Ubicaciones reordenadas."

#: ../cog_manager.py:282
msgid "That path does not exist."
msgstr "Ubicación inexistente."

#: ../cog_manager.py:286
msgid "The bot will install new cogs to the `{}` directory."
msgstr "El bot instalará nuevos cogs en el directorio `{}`."

#: ../core_commands.py:35
msgid "No module by that name was found in any cog path."
msgstr "No existe módulo con ese nombre en ninguna ubicación."

#: ../core_commands.py:43
msgid "Failed to load package. Check your console or logs for details."
msgstr "Error cargando paquete. Revisar consola o bitácora para más detalles."

#: ../core_commands.py:47 ../core_commands.py:56 ../core_commands.py:76
#: ../core_commands.py:151 ../core_commands.py:212 ../core_commands.py:226
msgid "Done."
msgstr "Terminado."

#: ../core_commands.py:58
msgid "That extension is not loaded."
msgstr "Esa extensión no está cargada."

#: ../core_commands.py:71
msgid "Failed to reload package. Check your console or logs for details."
msgstr ""
"Error recargando paquete. Revisar consola o bitácora para más detalles."

#: ../core_commands.py:86
msgid "Shutting down... "
msgstr "Apagando..."

#: ../core_commands.py:123
msgid "The admin role for this guild has been set."
msgstr "El rol de administrador para este gremio ha sido configurado."

#: ../core_commands.py:131
msgid "The mod role for this guild has been set."
msgstr "El rol de moderador para este gremio ha sido configurado."

#: ../core_commands.py:145
msgid ""
"Failed. Remember that you can edit my avatar up to two times a hour. The URL "
"must be a direct link to a JPG / PNG."
msgstr ""
"Error. Recordar que sólo se puede editar el avatar hasta dos veces por hora. "
"La URL debe ser un enlace directo a un JPG o PNG."

#: ../core_commands.py:149
msgid "JPG / PNG format only."
msgstr "Únicamente formatos JPG o PNG."

#: ../core_commands.py:161
msgid "Game set."
msgstr "Juego configurado."

#: ../core_commands.py:190
msgid "Status changed to %s."
msgstr "Estado cambiado a %s."

#: ../core_commands.py:221
msgid ""
"Failed to change name. Remember that you can only do it up to 2 times an "
"hour. Use nicknames if you need frequent changes. `{}set nickname`"
msgstr ""
"Error cambiando nombre. Recordar que sólo se puede hacer hasta dos veces por "
"hora. Usar sobrenombres si cambios frecuentes son necesarios. `{}set "
"nickname`"

#: ../core_commands.py:236
msgid "I lack the permissions to change my own nickname."
msgstr "Carezco de permisos para cambiar mi sobrenombre."

#: ../core_commands.py:250 ../core_commands.py:263
msgid "Prefix set."
msgstr "Prefijo configurado."

#: ../core_commands.py:259
msgid "Guild prefixes have been reset."
msgstr "Prefijos de gremio han sido restaurados."

#: ../core_commands.py:282
msgid ""
"\n"
"Verification token:"
msgstr ""
"\n"
"Ficha de verificación:"

#: ../core_commands.py:285
msgid "Remember:\n"
msgstr "Recordar:\n"

#: ../core_commands.py:288
msgid ""
"I have printed a one-time token in the console. Copy and paste it here to "
"confirm you are the owner."
msgstr ""
"He impreso una ficha única en la consola. Copiar y pegarlo aca para "
"confirmar propiedad."

#: ../core_commands.py:296
msgid "The set owner request has timed out."
msgstr "Tiempo de espera para configuración de dueño ha terminado."

#: ../core_commands.py:302
msgid "You have been set as owner."
msgstr "Has sido configurado como dueño."

#: ../core_commands.py:304
msgid "Invalid token."
msgstr "Ficha inválida."

#: ../core_commands.py:313
msgid "Locale has been set."
msgstr "Localización configurada."

#: ../core_commands.py:323
msgid "User ID: %s"
msgstr "ID de usuario: %s"

#: ../core_commands.py:326
msgid "through DM"
msgstr "a través de mensaje privado"

#: ../core_commands.py:328
msgid "from {}"
msgstr "de {}"

#: ../core_commands.py:329
msgid " | Server ID: %s"
msgstr " | ID de servidor: %s"

#: ../core_commands.py:337
msgid "Use `{}dm {} <text>` to reply to this user"
msgstr "Utilizar `{}dm {} <texto>` para responder a este usuario"

#: ../core_commands.py:345
msgid "Sent by {} {}"
msgstr "Enviado por {} {}"

#: ../core_commands.py:357
msgid "I cannot send your message, I'm unable to find my owner... *sigh*"
msgstr "Error enviando mensaje, no encuentro a mi dueño... *suspiro*"

#: ../core_commands.py:360
msgid "I'm unable to deliver your message. Sorry."
msgstr "No puedo enviar tu mensaje, perdón."

#: ../core_commands.py:362
msgid "Your message has been sent."
msgstr "Tu mensaje ha sido enviado."

#: ../core_commands.py:376
msgid ""
"Invalid ID or user not found. You can only send messages to people I share a "
"server with."
msgstr ""
"ID inválido o usuario no encontrad. Solo puedes enviar mensajes a personas "
"con quienes compartes un servidor."

#: ../core_commands.py:382
msgid "Owner of %s"
msgstr "Dueño de %s"

#: ../core_commands.py:385
msgid "You can reply to this message with %scontact"
msgstr "Puedes contestar este mensaje utilizando %scontact"

#: ../core_commands.py:395
msgid "Sorry, I couldn't deliver your message to %s"
msgstr "Perdón, no pude enviar tu mensaje para %s"

#: ../core_commands.py:398
msgid "Message delivered to %s"
msgstr "Mensaje enviado a %s"

#: ../dev_commands.py:165
msgid "Already running a REPL session in this channel. Exit it with `quit`."
msgstr "Una sesión de REPL ya esta activa en este canal. Salir con `quit`."

#: ../dev_commands.py:169
msgid "Enter code to execute or evaluate. `exit()` or `quit` to exit."
msgstr "Ingresar codigo a ejecutar o evaluar. `exit()` o `quit` para salir."

#: ../dev_commands.py:234
msgid "Unexpected error: `{}`"
msgstr "Error inesperado: `{}`"
